api/lib/infrastructure/repositories/session-for-monitoring-repository.js

```diff
@@ -0,0 +1,42 @@
+const { knex } = require('../../../db/knex-database-connection');
+const { NotFoundError } = require('../../domain/errors');
+const CertificationCandidateForMonitoring = require('../../domain/models/CertificationCandidateForMonitoring');
+const SessionForMonitoring = require('../../domain/models/SessionForMonitoring');
+
+module.exports = {
+  async get(idSession) {
+    const results = await knex
+      .select({
+        id: 'sessions.id',
+        date: 'sessions.date',
+        time: 'sessions.time',
+        room: 'sessions.room',
+        examiner: 'sessions.examiner',
+        certificationCenterName: 'certification-centers.name',
+        certificationCandidates: knex.raw(`
+          json_agg("certification-candidates".* order by lower("lastName"), lower("firstName"))
+      `),
+      })
+      .from('sessions')
+      .leftJoin('certification-candidates', 'certification-candidates.sessionId', 'sessions.id')
+      .leftJoin('certification-centers', 'certification-centers.id', 'sessions.certificationCenterId')
+      .groupBy('sessions.id', 'certification-centers.id')
+      .where({ 'sessions.id': idSession })
+      .first();
+    if (!results) {
+      throw new NotFoundError("La session n'existe pas");
+    }
+    return _toDomain(results);
+  },
+};
+
+function _toDomain(results) {
+  const toDomainCertificationCandidates = results.certificationCandidates.map((candidate) => {
```

Comment:
Proposition : juste `certificationCandidates` ?
```suggestion
  const certificationCandidates = results.certificationCandidates.map((candidate) => {
```